packages/tasks/src/model-libraries.ts

```diff
@@ -408,6 +408,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"tokenizer.model"`,
 	},
+	"refiners": {
+		prettyLabel: "Refiners",
+		repoName: "refiners",
+		repoUrl: "https://github.com/finegrain-ai/refiners",
+		docsUrl: "https://refine.rs/",
+		filter: false,
+		countDownloads: `path_extension:"safetensors"`,
```

Comment:
For what I can see in [juggernaut.reborn](https://huggingface.co/refiners/juggernaut.reborn/tree/main), [refiners/juggernaut.aftermath](https://huggingface.co/refiners/juggernaut.aftermath/tree/main) and [refiners/juggernaut.inpainting](https://huggingface.co/refiners/juggernaut.inpainting/tree/main), the model repos always contain the same three safetensors files. When using the `refiners` library, which file is necessarily downloaded? Is it the three of them? If yes, the risk is to count 3x the number of downloads hence artificially boosting the counter without relevant usage. What we suggest to do in this case is to count downloads only on 1 file, for example `countDownloads: `path:"text_encoder.safetensors"`,`. WDYT?

Reply:
Hi @Wauplin!
:thinking: Thanks I foresee that. I don't think we plan on supporting an "automagic" method like diffusers' ``

Reply:
Sounds good! Splitting repos is usually better to document things independently + monitor usage. Just let me know then :)

Reply:
Hi @Wauplin, I finally found the time to split the models into their own repo!
Should I squash the commits of this PR?

Reply:
Thanks @Laurent2916! This is much better like this :) Now [all refiners models](https://huggingface.co/models?other=refiners) have a `model.safetensors` file so let's count downloads on this file only.

```suggestion
		countDownloads: `path:"model.safetensors"`,
```